src/boot/bootctl-status.c

```diff
@@ -163,24 +253,51 @@ static int status_variables(void) {
         else if (n_order < 0)
                 return log_error_errno(n_order, "Failed to read EFI boot order: %m");
 
-        /* print entries in BootOrder first */
-        for (int i = 0; i < n_order; i++)
-                (void) print_efi_option(order[i], &n_printed, /* in_order= */ true);
+        /* print entries in BootOrder first or add to JsonVariant for Json output */
+        if (arg_json_format_flags != JSON_FORMAT_OFF) {
+                for (int i = 0; i < n_order; i++)
+                        (void) print_efi_option(order[i], &n_printed, /* in_order= */ true, &array);
 
-        /* print remaining entries */
-        for (int i = 0; i < n_options; i++) {
-                for (int j = 0; j < n_order; j++)
-                        if (options[i] == order[j])
-                                goto next_option;
+                /* print remaining entries */
+                for (int i = 0; i < n_options; i++) {
+                        for (int j = 0; j < n_order; j++)
+                                if (options[i] == order[j])
+                                        goto next_option_json;
 
-                (void) print_efi_option(options[i], &n_printed, /* in_order= */ false);
+                        (void) print_efi_option(options[i], &n_printed, /* in_order= */ false, &array);
+                next_option_json:
+                        continue;
+                }
 
-        next_option:
-                continue;
-        }
+                r = json_variant_merge_objectb(v, JSON_BUILD_OBJECT(
+                        JSON_BUILD_PAIR("Boot Loaders Listed in EFI Variables", JSON_BUILD_VARIANT(array))));
```

Comment:
this too is just a phrase as a json key, the point of json is machine-readable output

Reply:
Thanks, I understood this. Since it is json format, the key should not have space.